pkg/commands/policy/enforce.go

```diff
@@ -121,10 +144,19 @@ func (c *EnforceCommand) Process(ctx context.Context) error {
 			continue
 		}
 
+		_, _ = fmt.Fprintf(&b, "#### %s\n", k)
```

Comment:
nit: you dont need the `_` if your dropping everything anyways

throughout

```suggestion
		fmt.Fprintf(&b, "#### %s\n", k)
```

Reply:
is it resolved tho?